### PR TITLE
Changed the database

### PR DESCRIPTION
**Pull request by Rick0317**

### CODE REVIEW

File: database.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 0, Name, email, password, age, bio, gender, orientation, location_1: location_2, hobbies, socialMedia, likes, preferredAge, preferredGender, preferredLocation
 1, Name, email@, null, 20, bio, male, straight, 20.0: 10.0, play soccer: play violin, null, null, 20, male
-2, Taka, taka@mail, 123, 20, bio, male, straight, null, soccer: music, socialMedia, null, 20, male, null
-3, Name, email2, password, age, bio, gender, orientation, location, hobbies, socialMedia, likes, preferredAge, preferredGender, preferredLocation, -1
\ No newline at end of file
+2, Taka, taka@mail, null, 20, bio, male, straight, null, soccer: music, socialMedia, null, 20, male, null
+3, Name, emails, password, age, bio, gender, orientation, location, hobbies, socialMedia, likes, preferredAge, preferredGender, preferredLocation, -1
+
```
Comment (a11309) [inline database.txt:5]: I think you may need to delete the empty line

**Comment by Rick0317** [comment]
Right, I'll do it

**Review by a11309 — DISMISSED** [review]

**Review by etheliachoi — DISMISSED** [review]

**Comment by etheliachoi** [comment]
The line in DataFetchControl is reverted to the old one we used to have, correct?

**Review by a11309 — APPROVED** [review]

**Review by SiyounKim — APPROVED** [review]

**Comment by Rick0317** [comment]
I think this pull request doesn't change much rather it might changes the database.txt and might make some conflicts with codes in your local branch so I'll close this

**Pull request closed without merging**